### PR TITLE
#8617 Branding customization docs refactoring

### LINKED ISSUE #8617: Documentation: Refactor Branding Your Installation > Custom Header
As discussed in issue https://github.com/IQSS/dataverse/issues/8591, Branding Your Installation > Custom Header section in the documentation should be refactored to further explain what changes can be made and what to expect after customizing the page header.

### PR DESCRIPTION
**Pull request by j-n-c**
**What this PR does / why we need it**:

Refactor Branding customization docs:

- Make it a living document
- Set expectations on what will change by applying customization
- Include information about available navbar features

**Which issue(s) this PR closes**:

- Closes #8617 

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

File: doc/sphinx-guides/source/installation/config.rst [hunk 1]
```diff
@@ -2613,3 +2687,6 @@ To override the default list with Docs, Data, Code, and Workflow:
 To remove the override and go back to the default list:
 
 ``curl -X PUT -d '' http://localhost:8080/api/admin/settings/:FileCategories``
+
+.. |dvPageBlocks| image:: ./img/dvBrandingCustBlocks.png
```
Comment (pdurbin) [inline doc/sphinx-guides/source/installation/config.rst:2691]: @j-n-c I'm working on a more complete review, but a quick question for you. How did you create this PNG file? What if we want to edit it in the future? Is it possible to get the source of the image? I'm thinking about an SVG file, for example. Something we can edit. Thanks.

**Review by pdurbin — COMMENTED** [review]
Quick question

File: doc/sphinx-guides/source/installation/config.rst (hunk 1 above)
Comment (poikilotherm) [reply doc/sphinx-guides/source/installation/config.rst:2691]: **nitpick**: This appears in the [preview](https://dataverse-guide--8638.org.readthedocs.build/en/8638/installation/config.html#branding-your-installation), but shouldn't the image be referenced at the place of use? Scrolling all the way down seems cumbersome.

Comment (j-n-c) [reply doc/sphinx-guides/source/installation/config.rst:2691]: @pdurbin I have used [drawio](https://app.diagrams.net/) to create the image. You can download the software and run it offline or use the online editor.
Here is the editable file: [dv.zip](https://github.com/IQSS/dataverse/files/8609421/dv.zip)
.

Comment (pdurbin) [reply doc/sphinx-guides/source/installation/config.rst:2691]: @j-n-c thanks! The "dv.drawio" is only 30 KB so I'll probably just commit it to my branch (I'm planning on making a pull request for you to review).

I was able to import it into https://app.diagrams.net without any trouble. Here's a screenshot:

Screen Shot 2022-05-03 at 10 11 16 AM

@poikilotherm I guess it's personal preference. If you look at the README you'll see a long list of links at the bottom that are used throughout. I see this as similar. Put all that junk at the bottom.

**Review by pdurbin — APPROVED** [review]
Thanks for kicking off this pull request @j-n-c !I hope you don't mind that I pushed directly to your branch. 

Hopefully my changes help with clarity, etc.

@kcondon the interaction between :SystemEmail and :InstallationName wasn't very well specified before so I took a whack at it.

File: doc/sphinx-guides/source/installation/config.rst (hunk 1 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/config.rst:2723]: > nitpick: This appears in the [preview](https://dataverse-guide--8638.org.readthedocs.build/en/8638/installation/config.html#branding-your-installation), but shouldn't the image be referenced at the place of use? Scrolling all the way down (2000 lines) seems cumbersome.

@poikilotherm In https://github.com/ErykKul/dataverse/pull/1 I said "you were right". 😄 

People just want to add database settings to the bottom of the config page, as they should. Don't make me think!

**Pull request merged**